pina/condition/__init__.py

```diff
@@ -4,9 +4,15 @@
     "DomainEquationCondition",
     "InputPointsEquationCondition",
     "InputOutputPointsCondition",
+    "GraphInputOutputCondition",
+    "GraphDataCondition",
+    "GraphInputEquationCondition",
```

Comment:
is this really useful? I think this is useful only if you want to differentiate wrt graph tensors (node features, or others). Is it correct?